lib/views/options.html

```diff
@@ -287,11 +286,41 @@
+                      </label>
+                    </div>
+                    <div class="form-group">
+                      <label>
+                          <span id="lang_volume" class="locale">Default Volume Level</span>
+                          <input id="volume" ng-model="opctrl.volume" ng-change="opctrl.updatePlaybackOptions()" type="range" min="0.2" max="1.0" value="0.5" step='0.1'></input>
+                      </label>
+                    </div>
+                    <div class="form-group">
+                      <label>
+                          <span id="lang_rate" class="locale">Speech rate</span>
+                          <input id="rate" ng-model="opctrl.rate" ng-change="opctrl.updatePlaybackOptions()" type="range"  min="0.5" max="4.0" value="1.0" step="0.1"></input>
+                      </label>
+                    </div>
+                    <div class="form-group">
+                      <label>
+                          <span id="lang_pitch" class="locale">Pitch</span>
+                          <input id="pitch" type="range" ng-change="opctrl.updatePlaybackOptions()" ng-model="opctrl.pitch" min="0.0" max="2.0" value="1.0" step="0.2"></input>
+                      </label>
+                    </div>
+                  </form>
+                </div>
+              </div>
             </div>
           </div>
-        </div>
```

Comment:
@rohankatyal29 @Ceilican this `</div>` should not be removed. It breaks the contribute page.